transaction/issuance.go

```diff
@@ -125,6 +125,11 @@ func NewTxIssuance(
 	if err != nil {
 		return nil, err
 	}
+
+	if assetAmount == 0 {
+		confAssetAmount = []byte{0}
+	}
+
 	confTokenAmount, err := toConfidentialTokenAmount(
 		tokenAmount,
 	)
```

Comment:
Couldnt happen to have in some cases also the token amount = 0? Ie. an Issuance without reissuance token (ofc in that case asset amount MUST be > 0 otherwise makes no sense)

Reply:
`toConfidentialTokenAmount()` already takes care of returning `[]byte(0x00)` in case the token amount is 0.
In fact, we can make it more generic so that we can use it also for asset amount ([ref to comment](https://github.com/vulpemventures/go-elements/pull/194#discussion_r882575499))

Reply:
Done